src/output.cpp

```diff
@@ -475,20 +478,22 @@ void Output::map_stats_contents(
     const BpfMap &map,
     uint32_t top,
     uint32_t div,
-    const std::map<std::vector<uint8_t>, std::vector<int64_t>> &values_by_key,
-    const std::vector<std::pair<std::vector<uint8_t>, int64_t>>
-        &total_counts_by_key) const
+    const std::vector<std::pair<std::vector<uint8_t>, std::vector<uint8_t>>>
+        &values_by_key) const
 {
   const auto &map_type = bpftrace.resources.maps_info.at(map.name()).value_type;
   uint32_t i = 0;
+  size_t total = values_by_key.size();
   bool first = true;
-  for (auto &key_count : total_counts_by_key) {
-    auto &key = key_count.first;
-    auto &value = values_by_key.at(key);
 
-    if (map_type.IsAvgTy() && top && values_by_key.size() > top &&
-        i++ < (values_by_key.size() - top))
-      continue;
+  for (auto &pair : values_by_key) {
+    auto key = pair.first;
+    auto value = pair.second;
```

Comment:
```suggestion
  for (auto &[key, value] : values_by_key) {
```
Can be shortened to a single line. Also I think the current code is making copies of the `key` and `value` vectors (`auto` instead of `auto&`)